lib/ferrum/page/screenshot.rb

```diff
@@ -123,9 +123,21 @@ def screenshot_options(path = nil, format: nil, scale: 1.0, **opts)
           options[:clip].merge!(scale: scale)
         end
 
+        if validate_background_rgba_color(opts[:background_rgba_color])
+          raise ArgumentError, "Specify :background_rgba_color as [R,G,B,A] array"
+        end
+
         options
       end
 
+      def validate_background_rgba_color(option)
```

Comment:
I don't feel like this code belongs to screenshot. Anonymous array is definitely a dynamic typing approach but maybe if we introduce something like

```ruby
browser.screenshot(background_rgba_color: [0, 0, 0, 0.0])

browser.screenshot(background_color: RGBA.new(0, 0, 0, 0.0))
```

we could hide validations in there as well as make this variable sound right